packages/radix-ui-themes/src/components/skeleton.css

```diff
@@ -0,0 +1,78 @@
+:where(.rt-Skeleton) {
+  border-radius: var(--radius-1);
+}
+
+.rt-Skeleton {
+  animation: rt-pulse 1000ms infinite alternate-reverse !important;
+  background-image: none !important;
+  border: none !important;
+  box-shadow: none !important;
+  color: transparent !important;
+  outline: none !important;
+  pointer-events: none !important;
+  user-select: none !important;
+  cursor: default !important;
+  box-decoration-break: clone !important;
+}
+
+/*
+ * Take font differences out of equation for inline Skeletons that just wrap text.
+ * This makes the dimensions consistent between the following non-Skeleton and Skeleton text:
+ * ```
+ * <Text>Example text</Text>
+ *
+ * <Text>
+ *   <Skeleton>Example text</Skeleton>
+ * </Text>
+ * ```
+ *
+ * Will NOT collapse Skeletons as the browsers use line-height on inline elements just for alignment.
+ */
+.rt-Skeleton[data-inline-skeleton] {
+  line-height: 0;
+  /* Font metrics on Arial yield much nicer height for the text background bounding box */
+  font-family: Arial, sans-serif !important;
+}
+
+.rt-Skeleton:autofill {
+  -webkit-text-fill-color: transparent !important;
+  box-shadow: inset 0 0 0 100px var(--gray-a3), inset 0 0 0 100px var(--color-background) !important;
+}
```

Comment:
- We should check if this is actually needed, this style was authored for the pre-RT Text Field
- If still needed, we can use the newer technique for removing the native autofill colors that we use in the Text Field itself

Reply:
How would this state even trigger? You'd have to be auto-filling something and THEN it goes skeleton. Seems unlikely no?

Reply:
Browsers will autofill forms you had submitted before on page load, check this out:

https://github.com/radix-ui/themes/assets/8441036/6760de0a-2f1a-44ac-be91-a62b5dc7bf3b

Reply:
I might need help testing this out, I can't get it to even auto-fill a whole email/password form in Chrome.

https://github.com/radix-ui/themes/assets/1539897/47b1583b-5bb1-4e1b-bef8-cfcdee5d4f99

Reply:
@benoitgrelard I think you need a form that you'd submit at least once to get that
Pushed a `test-skeleton` page to your branch with that and tested:

https://github.com/radix-ui/themes/assets/8441036/5806c1d2-7c10-428e-848f-cda320002d16

Looks like the workaround for autofill is still needed; and the current technique does have a cosmetic problem, you can see that the browser’s autofill kicks in half a second after you load the page and makes the animated elements feel out of sync.

Here’s what it looks like if we don't handle autofill:

https://github.com/radix-ui/themes/assets/8441036/31e98b0b-e6b5-443a-a905-e70500ce320b